src/model/Task.model.js

```diff
@@ -0,0 +1,16 @@
+export default class Task {
+  constructor (id, state, latestMessage, depth, jobs) {
+    this.id = id
+    this.state = state
+    this.latestMessage = latestMessage
+    this.depth = depth
+    this.jobs = jobs
+  }
+
+  get jobId () {
+    if (this.jobs !== undefined && this.jobs.length > 0) {
+      return this.jobs[0].batchSysJobId
```

Comment:
Should go for latest `this.jobs[jobs.length -1]`.

Reply:
Found another way. Done with [slice](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice) `this.jobs.slice(-1)[0]`